cumulusci/tasks/robotframework/robotframework.py

```diff
@@ -98,15 +110,38 @@ def _init_options(self, kwargs):
 
         self.options.setdefault("sources", [])
 
+    def determine_output_directory(self):
+
+        # This code is not clear to me so I'm a bit afraid to change it
+        def relativize_path(output_dir):
+            return os.path.relpath(
+                os.path.join(self.working_path, output_dir), os.getcwd()
+            )
+
+        output_dir = None
+
+        if os.environ.get("CUMULUSCI_ROBOT_OUTPUT_DIR"):
+            output_dir = relativize_path(os.environ.get("CUMULUSCI_ROBOT_OUTPUT_DIR"))
```

Comment:
Brandon is currently working on adjusting the way that MetaCI imports robot results so that it will be aware of the task's outputdir option (https://github.com/SFDO-Tooling/MetaCI/pull/1786). So it won't be necessary to change it for MetaCI, and won't be necessary to have this way to override the default.

Reply:
Does the rest make sense to you?

Reply:
Yes, the rest does, at least at first glance.